ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/SandboxOnXRunner.scala

```diff
@@ -81,7 +83,7 @@ object SandboxOnXRunner {
 
       // Start the ledger
       (participantId, dataSource, participantConfig) <- combinedParticipant(config)
-      _ <- buildLedger(
+      (apiServer, _, _) <- buildLedger(
```

Comment:
https://discuss.daml.com/t/pattern-always-adds-withfilter-scala-ritual/1463

Reply:
I think I don't need to expose this tuple anymore, will just remove it

Reply:
in fact I still need it for sandbox classic, so will apply the fix, thanks!